projects/storefrontapp-e2e-cypress/cypress.json

```diff
@@ -1,15 +1,15 @@
 {
-  "defaultCommandTimeout": 10000,
+  "defaultCommandTimeout": 30000,
   "baseUrl": "http://localhost:4200",
-  "requestTimeout": 15000,
+  "requestTimeout": 45000,
   "retries": {
-    "runMode": 2
+    "runMode": 3
   },
   "videoUploadOnPasses": false,
   "env": {
     "CLIENT_ID": "mobile_android",
     "CLIENT_SECRET": "secret",
-    "API_URL": "https://40.76.109.9:9002",
+    "API_URL": "https://20.83.184.244:9002",
```

Comment:
I think the default url of this file should point to dev.  It was set to dev in the past to keep default traffic away from the CI backends.

Reply:
Agree. Updated by mistake. Will revert.